src/test/java/com/puppycrawl/tools/checkstyle/api/FullIdentTest.java

```diff
@@ -210,6 +210,42 @@ private static FullIdent prepareFullIdentWithCoordinates(int columnNo, int lineN
         return FullIdent.createFullIdent(ast);
     }
 
+    @Test
+    public void testNewTypeTag() throws Exception {
+        final DetailAstImpl parent = new DetailAstImpl();
+        parent.setType(TokenTypes.TYPE);
```

Comment:
we banned pure junit test methods as they does not contribute to quality as much as we need.


can you make coverage as in testCreateFullIdentBelow2  of this class?

Reply:
done